public/index.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<link rel="stylesheet" type="text/css" href="css/main.css">
+	<!-- Load the Paper.js library -->
+	<script src="https://ajax.googleapis.com/ajax/libs/jquery/2.2.4/jquery.min.js"></script>
+	<script type="text/javascript" src="/js/paper.js"></script>
+	<script src="https://cdn.socket.io/socket.io-1.2.0.js"></script>
```

Comment:
totally agree. I used CDN to make sure server-to-client communication works. we are going to place the client code in a separate file anyway. we will require the socket.io-client from there